src/components/views/SSITabView/index.tsx

```diff
@@ -7,17 +8,18 @@ import {ITabRoute, ITabViewRoute} from '../../../types';
 import SSITabViewHeader from './SSITabViewHeader';
 
 export interface IProps {
+  style?: ViewStyle;
   routes: Array<ITabViewRoute>;
 }
 
 const SSITabView: FC<IProps> = (props: IProps): JSX.Element => {
   const [index, setIndex] = React.useState(0);
   const [routes] = React.useState(props.routes);
-
+  const style = props.style;
```

Comment:
i rather deconstruct props where possible:

const { style } = props

then you can use style where you want. you can then also try to at routes to it, but it shares the name of the state

Reply:
remvoed this styling